test/integration/manifest_int_test.go

```diff
@@ -50,6 +53,54 @@ func (suite *ManifestIntegrationTestSuite) TestManifest_JSON() {
 	cp.Expect(`"requirements":`)
 }
 
+func (suite *ManifestIntegrationTestSuite) TestManifest_Advanced_Reqs() {
+	suite.OnlyRunForTags(tagsuite.Manifest)
+	ts := e2e.New(suite.T(), false)
+	defer ts.Close()
+
+	ts.LoginAsPersistentUser()
+
+	cp := ts.Spawn("config", "set", constants.OptinBuildscriptsConfig, "true")
+	cp.ExpectExitCode(0)
+
+	ts.PrepareProject("ActiveState/cli", "00000000-0000-0000-0000-000000000000")
```

Comment:
nit: We have https://github.com/ActiveState/cli/blob/c2ddc673aec0624235df463093ff44b785cc954f/internal/testhelpers/e2e/constants.go#L5 for this.